src/main/pg/autopilot.h

```diff
@@ -29,10 +29,11 @@ typedef struct autopilotConfig_s {
     uint16_t hover_throttle;      // value used at the start of a rescue or position hold
     uint16_t throttle_min;
     uint16_t throttle_max;
-    uint8_t altitude_P;
-    uint8_t altitude_I;
-    uint8_t altitude_D;
-    uint8_t altitude_F;
+    uint16_t altitude_P;
```

Comment:
@ledvinap
Do you think this PR is useful, or should I close it?

Reply:
`src/main/pg/autopilot.c`:`30`:`PG_REGISTER_WITH_RESET_TEMPLATE(autopilotConfig_t, autopilotConfig, PG_AUTOPILOT, 2);` bump to 2

Reply:
> @ledvinap Do you think this PR is useful, or should I close it?

As written, it is interesting hack of PID output, but at minimum dubious from physical point of view. It has little to do with thrust linearization, it in effect decreases PID gains with larger error. From your testing, it helps with PID tuning - it may be because it actually improves PID, or because it makes PID parameters less sensitive to mistuning (more forgiving).

Can you try 0..1 normalization (to restore old PID parameter range) first?

Then you can try applying `sqrt` correctly (to get roughly linear response assuming quadratic rpm-throttle dependence).  Just take care to handle hover-throttle correctly

As @ctzsnooze noted, there is something called `pidApplyThrustLinearization`. It does apply some third-order function to throttle (https://www.wolframalpha.com/input?i=plot%5B%7Bx+*+%281+%2B+1.5+*%281+-+x%29+%5E+2%29%7D%2C%7Bx%2C0%2C1%7D%5D), you may try if for comparison.